### PR TITLE
docs(fix): reviewed block storage how-tos and tutos and added dates MTA-1639

### PR DESCRIPTION
**Pull request by nedaehm**

### CODE REVIEW

File: storage/block/api-cli/creating-a-volume.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ dates:
   ```
   $ curl -q \
     -H "X-Auth-Token: $SECRET_KEY" \
-    -H 'Content-Type: application/json'  \
+    -HContent-Type: application/json'  \
```
Comment (nedaehm) [inline storage/block/api-cli/creating-a-volume.mdx:40]:
```suggestion
    -HContent-Type: application/json'  \
```
```suggestion
    -H 'Content-Type: application/json'  \
```

File: storage/block/how-to/delete-a-volume.mdx [hunk 2]
```diff
@@ -33,8 +33,8 @@ You can only delete a Block volume when it is detached from the instance. See th
 5. Click **Delete**. A pop-up displays.
   <Lightbox src="scaleway-block-delete-volume.png" alt="" />
 
-5. Type `DELETE` to confirm.
-6. Click **Delete this volume** to remove the volume from your account.
+6. Type `DELETE` to confirm.
+7. Click **Delete this volume** to remove the volume from your account.
 	<Message type="important">
   All data on the volume will be removed when you delete it. Make sure to have a backup of your data before removing the volume.
```
Comment (nedaehm) [inline storage/block/how-to/delete-a-volume.mdx:39]:
```suggestion
  All data on the volume will be removed when you delete it. Make sure you have a backup of your data before removing the volume.
```

File: storage/block/api-cli/creating-a-volume.mdx [hunk 3]
```diff
@@ -21,7 +21,7 @@ dates:
 
 </Message>
 
-1. Create a block volume from the API by launching the following command:
+1. Create a block volume from your API by launching the following command:
```
Comment (bene2k1) [inline storage/block/api-cli/creating-a-volume.mdx:24]: Pourquoi your?

Comment (RoRoJ) [reply storage/block/api-cli/creating-a-volume.mdx:24]: agree, I think "the" makes more sense here :-)

Comment (nedaehm) [inline storage/block/api-cli/creating-a-volume.mdx:24]:
```suggestion
1. Create a block volume from the API by launching the following command:
```

Comment (nedaehm) [reply storage/block/api-cli/creating-a-volume.mdx]: Thank you, I changed it back to "the"!

**Review by RoRoJ — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

**Pull request merged**